lib/bundle/commands/check.rb

```diff
@@ -14,54 +16,75 @@ def reset!
         Bundle::BrewServices.reset!
       end
 
+      def exit_on_first_error?
+        Bundle::Checker.exit_on_first_error?
+      end
+
+      def output_errors?
+        ARGV.include?("--verbose")
```

Comment:
Feels weird to have `exit_on_first_error?` call a method elsewhere and just be the inverse of this?

Reply:
It's more explicit that way, should another option (e.g. `--exit-on-error` or `--output-all-errors`) or other logic come into play. It separates the desire from the business logic.

The next step would be making `"--verbose"` a constant, haha.

Reply:
I think just using `ARGV.include?` everywhere is a bit more obvious. If we introduce another option I think refactoring can be done there otherwise: YAGNI 😉

Reply:
Personally I find this less rather than more readable; it's unclear to me that `exit_on_first_error` and `output_errors` are related due to the indirection.

Reply:
I moved them together into `Bundle::Checker` and the relationship is more apparent. However, I dislike them there because then calling them is `Bundle.checker.output_errors?` and that's long.

I'm gonna mess around with it a little more _after_ I refactor the brew/cask/mas checkers.